### PR TITLE
Exclude config-1.2.1 from eclipse classpath

### PR DESCRIPTION
**Pull request by kashike**
This fixes https://github.com/SpongePowered/SpongeForge/issues/404

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -79,6 +79,12 @@ shadowJar {
     }
 }
 
+eclipse.classpath.file.whenMerged {classpath ->
+    classpath.entries.removeAll(classpath.entries.findAll {
+        entry -> entry.kind == 'lib' && entry.path.endsWith('com.typesafe/config/1.2.1/f771f71fdae3df231bcd54d5ca2d57f0bf93f467/config-1.2.1.jar')
```
Comment (gabizou) [inline build.gradle]: And if this changes at all? How about something more generic where it's just a default search of `.contains('com.typesafe/config/1.2.1/')`?

Comment (kashike) [inline build.gradle]: Why would a released version's hash change?

Comment (gabizou) [inline build.gradle]: Surefire way to know that regardless of the hash that it's going to be excluded.

**Comment by JBYoshi** [comment]
Why was this closed?

**Pull request closed without merging**